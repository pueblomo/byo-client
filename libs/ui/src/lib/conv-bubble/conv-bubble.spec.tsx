import React from 'react';
import { render } from '@testing-library/react-native';

import ConvBubble from './conv-bubble';

describe('ConvBubble', () => {
  it('should render successfully', () => {
    const tree = render(
      <ConvBubble isSelf={false}> Render incomming message </ConvBubble>
    );
    expect(tree).toMatchSnapshot();
  });

  it('should render self successfully', () => {
    const tree = render(
      <ConvBubble isSelf={true}> Render incomming message </ConvBubble>
    );
    expect(tree).toMatchSnapshot();
  });
});
