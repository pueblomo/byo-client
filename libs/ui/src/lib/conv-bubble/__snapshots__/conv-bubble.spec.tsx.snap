// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConvBubble should render self successfully 1`] = `
<View
  style={
    Object {
      "shadowColor": "#171717",
      "shadowOffset": Object {
        "height": 3,
        "width": 0,
      },
      "shadowOpacity": 0.4,
      "shadowRadius": 2,
    }
  }
>
  <View
    collapsable={false}
    style={
      Object {
        "alignSelf": "flex-end",
        "backgroundColor": "#79C7C5",
        "borderRadius": 10,
        "marginBottom": 10,
        "marginLeft": 20,
        "marginRight": 20,
        "marginTop": 10,
        "maxWidth": "60%",
        "padding": 10,
        "transform": Array [
          Object {
            "translateY": -50,
          },
        ],
      }
    }
  >
    <Text
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
       Render incomming message 
    </Text>
  </View>
</View>
`;

exports[`ConvBubble should render successfully 1`] = `
<View
  style={
    Object {
      "shadowColor": "#171717",
      "shadowOffset": Object {
        "height": 3,
        "width": 0,
      },
      "shadowOpacity": 0.4,
      "shadowRadius": 2,
    }
  }
>
  <View
    collapsable={false}
    style={
      Object {
        "alignSelf": "flex-start",
        "backgroundColor": "#777777",
        "borderRadius": 10,
        "marginBottom": 10,
        "marginLeft": 20,
        "marginRight": 20,
        "marginTop": 10,
        "maxWidth": "60%",
        "padding": 10,
        "transform": Array [
          Object {
            "translateY": -50,
          },
        ],
      }
    }
  >
    <Text
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
       Render incomming message 
    </Text>
  </View>
</View>
`;
